packages/script-writer/src/prompts.ts

```diff
@@ -0,0 +1,37 @@
+import { trimText } from "@evo-ninja/agent-utils";
+
+export const INITIAL_PROMP = `You are an agent that writes JavaScript functions.\n` +
+`Before writing any code think step by step about what you want to implement.\n` +
+`Call the writeFunction function to submit the code of your JavaScript function.\n` +
+`If the first try doesn't succeed, try again. Do not create mock functionality.\n`;
+
+export const GOAL_PROMPT = (namespace: string, description: string, args: string) => 
+  `Your task is to write the body of an async JavaScript function.\nFunction namepace: "${namespace}"\nArguments: ${args}.\nDescription: "${description}"\n` +
+  `You must refer to function arguments as if they were locally defined variables, remember you're writing just the body of the function.\n` +
+  `Use only the function arguments above, do not add new ones.\n` +
+  `Since you are writing the body of the function, remember to use the return keyword if needed.\n` +
+  `When using libraries, use the require function to import them.\n` +
+  `Do not require libraries aside from 'fs' and 'axios'\n` +
+  `Do not use external APIs that require authentication or an API key.\n` + 
+  `Example function body:\n` +
+  `const fs = require('fs');\n` +
+  `return fs.readFileSync(path, encoding);\n`;
```

Comment:
I think it makes sense to also have the js-engine types & shims in this package, as it fits nicely within this package's responsibilities.

Reply:
Does the script writer even use the engine or shims at all? I don't think it does. It would be weird to put them there at this moment

Reply:
Ah true, good point, though it does make explicit reference to what supported packages (shims) exist.